youcan-shop/cod-theme-3
# YSHOP2-930: theme 3 > express checkout

## JIRA Ticket

[YSHOP2-930](https://youcanshop.atlassian.net/browse/YSHOP2-930)

## QA Steps

-  [ ]  In theme editor go to product page > single product section > express checkout bloc
check that from background color isn’t available as option
- [ ]  Check that the UI of the express checkout bloc is similar to new updates that mention in the ticket

## Note

Leave empty when you have nothing to say


[YSHOP2-930]: https://youcanshop.atlassian.net/browse/YSHOP2-930?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
